src/jquery.nepaliDatePicker.js

```diff
@@ -8,8 +8,8 @@
 var calendarFunctions = {};
 (function ($) {
     var calendarData = {
-        bsMonths: ["बैशाख", "जेठ", "असार", "सावन", "भदौ", "असोज", "कार्तिक", "मंसिर", "पौष", "माघ", "फागुन", "चैत"],
-        bsDays: ["आईत", "सोम", "मंगल", "बुध", "बिही", "शुक्र", "शनि"],
+        bsMonths: ["वैशाख", "जेठ", "असार", "श्रावण", "भदौ", "असोज", "कार्तिक", "मङ्सिर", "पौष", "माघ", "फागुन", "चैत"],
```

Comment:
@sagunraj Thank you for the PR. 

1. `मंसिर` is correct. 
2. `श्रावण` could be made `साउन` because most people say `साउन` when they speak.

https://en.wikipedia.org/wiki/Mangsir

Reply:
As far as I know, मं is pronounced as मम्. That's the reason behind using ङ् there.

Reply:
Your spelling seems to be correct but what I'm saying is let's go with what everyone else is using.

Take a look at these examples: 

1. https://nepalipatro.com.np/calendar/monthly/bs/2077/8
2. https://www.hamropatro.com/calendar/2077/8
3. http://nepcal.com/
4. https://en.wikipedia.org/wiki/Mangsir

Same for `साउन` because It is prevalent.

Reply:
I've updated साउन but in case of मङ्सिर, I don't think it's right to do something just because everyone is doing that. We have a responsibility to preserve our language and script.

Reply:
You are not wrong but the spelling `मंसिर` is not incorrect. `मं` is a nasal consonant. I'm not a Nepali guru by any means but `मंसिर` is pronounced as `mangsir` and not `mamsir`.

> Shirbindu (represented as U+0902 DEVANAGARI SIGN ANUSVARA in Unicode) indicates a nasal consonant preceding another consonant, e.g., Shirbindu in प्रशंसा /prasamsa/ indicates nasal consonant म /ma/ preceding the consonant स /sa/, and Shirbindu in अंक /anka/ indicates nasal consonant ङ /ŋa/ preceding the consonant क /ka/.

https://nepalilanguage.org/alphabet/

Hope it helps. Not these many sites will want to make a mistake.

Reply:
Okay, I'll update it. :) Thank you!

Reply:
Thanks, will merge it when I have the access.